app/models/user.rb

```diff
@@ -0,0 +1,32 @@
+class User < ApplicationRecord
+  has_secure_password
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
+  validates :name,
+            presence: true,
+            length: {maximum: Settings.development.user.MAX_NAME_LENGTH}
```

Comment:
```suggestion
            length: {maximum: Settings.user.max_name_length}
```
Tương tự cho biến ở dưới

Reply:
Em bỏ chữ development đi xong test lại thì bị lỗi này ạ 
<img width="2056" height="602" alt="image" src="https://github.com/user-attachments/assets/25b334b8-b571-44bd-8865-d2754979f6be" />